blocks/header-nav-chain-block/_index.scss

```diff
@@ -0,0 +1,262 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-header-nav-chain {
+	.nav-search {
+		input {
+			@include scss.block-components("header-nav-chain-nav-search-input");
+			@include scss.block-properties("header-nav-chain-nav-search-input");
+		}
+
+		&.open {
+			input {
+				@include scss.block-components("header-nav-chain-nav-search-input-open");
+				@include scss.block-properties("header-nav-chain-nav-search-input-open");
+			}
+
+			button {
+				@include scss.block-components("header-nav-chain-nav-search-button");
+				@include scss.block-properties("header-nav-chain-nav-search-button");
+			}
+		}
+
+		@include scss.block-components("header-nav-chain-nav-search");
+		@include scss.block-properties("header-nav-chain-nav-search");
+	}
+
+	&__flyout-overlay {
+		&::-webkit-scrollbar {
+			@include scss.block-components("header-nav-chain-flyout-overlay-scrollbar");
+			@include scss.block-properties("header-nav-chain-flyout-overlay-scrollbar");
+		}
+
+		&.open {
+			@include scss.block-components("header-nav-chain-flyout-overlay-open");
+			@include scss.block-properties("header-nav-chain-flyout-overlay-open");
+		}
+
+		&.closed {
+			.section-menu {
+				@include scss.block-components("header-nav-chain-flyout-overlay-closed-transition");
+				@include scss.block-properties("header-nav-chain-flyout-overlay-closed-transition");
+			}
+
+			@include scss.block-components("header-nav-chain-flyout-overlay-closed");
+			@include scss.block-properties("header-nav-chain-flyout-overlay-closed");
+		}
+
+		@include scss.block-components("header-nav-chain-flyout-overlay");
+		@include scss.block-properties("header-nav-chain-flyout-overlay");
+	}
+
+	&__flyout-nav-wrapper {
+		&.open {
+			@include scss.block-components("header-nav-chain-flyout-nav-wrapper-open");
+			@include scss.block-properties("header-nav-chain-flyout-nav-wrapper-open");
+		}
+
+		&.closed {
+			@include scss.block-components("header-nav-chain-flyout-nav-wrapper-closed");
+			@include scss.block-properties("header-nav-chain-flyout-nav-wrapper-closed");
+		}
+
+		.nav-widget {
```

Comment:
Looks like this classname snuck through without using the bem naming. Can this be updated to follow our convention? 

`				<div className="nav-widget" key={`${id}_${breakpoint}_${i}`}>` in widget-list.jsx

Reply:
Fixed